docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/ci-cd-runs/add-checkov.adoc

```diff
@@ -76,6 +76,46 @@ NOTE: See the Checkov https://www.checkov.io/2.Basics/CLI%20Command%20Reference.
 * To view connected repositories, in *Application Security* select *Home* > *Projects* > *CI/CD Runs* tab. 
 * To view scan results and resolve issues, in *Application Security* select *Home* > *Projects*. See xref:../../../risk-management/monitor-and-manage-code-build/monitor-and-manage-code-build.adoc[Monitor and Manage Code Build Issues] for more information  
 
+=== Other Configuration Settings
+
+In order to use Checkov with a proxy, set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables (some environments will use lowercase `http_proxy` and `https_proxy`). Example:
+
+* macOS
++
+[source.shell]
+----
+# Set proxy environment variables
```

Comment:
Question:
Maybe we should tell the customer always to put both lower and upper case? 
this case, he will cover all operation systems.

Reply:
Works for me. I will update.